packages/client/hmi-client/src/composables/usePDFViewerActions.ts

```diff
@@ -0,0 +1,82 @@
+import { PDFAnnotation, PDFPageScrollPosition } from '@/types/common';
+import { BBox } from '@/types/Types';
+import { ref } from 'vue';
+
+const HIGHLIGHT_COLOR = '#fc0';
```

Comment:
What about this one? `#EBF4F3`
https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L938C23-L938C30

Reply:
<img width="1398" alt="Screenshot 2025-04-03 at 10 53 58 AM" src="https://github.com/user-attachments/assets/6898be67-e693-4aa3-a2a6-6730e8d213cb" />
Updated